kong/pdk/service/request.lua

```diff
@@ -417,6 +419,49 @@ local function new(self)
   end
 
 
+  if ngx.config.subsystem == "http" then
```

Comment:
Why not support this in stream mode?
Should we provide a stub/error function in stream mode until if we are delaying the work?

Reply:
Talked with @thibaultcha, it should be alright to skip this for now until someone complaints (and it is not a silent failure)...